src/node.cc

```diff
@@ -3366,16 +3365,17 @@ void LoadEnvironment(Environment* env) {
     ReportException(env, try_catch);
     exit(10);
   }
+  // The bootstrap_node.js file returns a function 'f'
   CHECK(f_value->IsFunction());
   Local<Function> f = Local<Function>::Cast(f_value);
 
   // Now we call 'f' with the 'process' variable that we've built up with
-  // all our bindings. Inside node.js we'll take care of assigning things to
-  // their places.
+  // all our bindings. Inside bootstrap_node.js we'll take care of
```

Comment:
Sorry, I might have misunderstood @mscdex [comment](https://github.com/nodejs/node/pull/8092#discussion-diff-74685414) about the full path as I removed it. I'm happy to change this, but at this point I'm not exactly sure if I should add `internal` to all references or not.